web/react/components/channel_header.jsx

```diff
@@ -194,6 +194,17 @@ export default class ChannelHeader extends React.Component {
             );
         }
 
+        const popoverListMembers = [];
+        if (!isDirect) {
+            popoverListMembers.push(
+                <PopoverListMembers
+                    members={this.state.users}
+                    memberCount={this.state.userCount}
+                    channelId={channel.id}
+                />
```

Comment:
Can you change `popoverListMembers` over to just a React node? If there's only ever going to be one element then there's no need for an array and React will throw a warning unless each array element has a key prop.

So maybe do something like this:

```
let popoverListMembers;
if (!isDirect) {
     popoverListMembers = (
         <PopoverListMembers
             members={this.state.users}
             memberCount={this.state.userCount}
             channelId={channel.id}
         />
    );
}
```